src/mono/wasm/runtime/imports.ts

```diff
@@ -51,7 +51,6 @@ export function set_emscripten_entrypoint(
 const initialRuntimeHelpers: Partial<RuntimeHelpers> =
 {
     javaScriptExports: {} as any,
-    mono_wasm_load_runtime_done: false,
     mono_wasm_bindings_is_ready: false,
```

Comment:
Do we still need this variable when it's not guarding `bindings_init` anymore?

Reply:
The question is about `mono_wasm_bindings_is_ready` right ?
That is still useful in making sure that interop API is not called before it was initialized.